build.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const fs = require('fs');
+const http = require('follow-redirects').http;
+const archiver = require('archiver');
+
+var options = {
+    host: 'uplift.shipit.staging.mozilla-releng.net',
+    path: '/coverage/supported_extensions',
+    method: 'GET',
+    headers: {"Content-Type": "text/plain"}
+}
+
+
+var req = http.request(options, (res) => {
+
+	var data = '';
+    
+    res.on('data', (chunk) => {
+        data += chunk;
+    });
+
+    res.on('end', () => {
+
+        fs.writeFile("supported_extensions.js", "const SUPPORTED_EXTENSIONS = " + data, (e) => {
+	    	if(e) {
+	    		console.log(e.message);
+	    	}
+		});
+
+		var excludeFiles = [".gitignore", ".travis.yml", "LICENSE", "README.md", "package.json",
+			  "package-lock.json", "requirements.txt", "setup.cfg", "build.js", "build.py"];
+		
+		var resultFiles = [];
+
+		fs.readdir(".", (err, files) => {
+			files.forEach(file => {
```

Comment:
Instead of `forEach`, you could use `filter`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Then pass the result to `makeZip`.